script_library/requirements.txt

```diff
@@ -1,6 +1,6 @@
 ansible==2.7.8
 netaddr==0.7.19
-jmespath==0.9.3
+jmespath>=0.9.3
```

Comment:
????
Any reason why this, why is not mentioned and why are we unfreezing a known version?

Reply:
no reason. just by accident :-)

Reply:
fixed